modules/cloudaccess/examples/complete/aws-to-azure.tfvars.sample

```diff
@@ -1,13 +1,13 @@
 # Resource suffix is optional, specify this if you need to keep role definitions unique amongst
 # multiple workspaces using the same subscription.
 # e.g. resource_suffix = "-app1-nonprod"
-resource_suffix                       = ""
-
+resource_suffix                 = ""
 wayfinder_identity_aws_role_arn = "arn:aws:iam::123456789012:role/wf-cloudidentity-aws"
 wayfinder_identity_aws_issuer   = "https://oidc.eks.eu-west-2.amazonaws.com/id/ABCDF8D2EF45CE1A6FD96E2781310123"
 wayfinder_identity_aws_subject  = "system:serviceaccount:wayfinder:wayfinder-admin"
-
-enable_cluster_manager                = true
-enable_dns_zone_manager               = true
-enable_network_manager                = true
-enable_cloud_info                     = false
+enable_cluster_manager          = true
+enable_dns_zone_manager         = true
+enable_network_manager          = true
+enable_cloud_info               = false
+from_aws                        = true
+from_azure                      = false
```

Comment:
minor style issue - i'd put these along with the AWS role ARN as they're 'contextually related'
also: I think we need to go back to suggesting a resource_suffix on azure as the role definition names need to be tenant-wide unique, but i'll raise a separate PR for that.